src/App.tsx

```diff
@@ -13,8 +13,22 @@ export class App extends React.Component<{}, State> {
     movies: moviesFromServer,
   };
 
-  addMovie = (/* movie: Movie */) => {
-    // put your code here
+  adder = (title: string,
```

Comment:
and better pass a whole movie, not every field

```suggestion
  addMovie = (title: string,
```